src/core/packages/saved-objects/server/src/mapping_definition.ts

```diff
@@ -74,9 +75,9 @@ export type SavedObjectsFieldMapping = EsMappingProperty &
      * index mappings.
      *
      * Note: To limit the number of mapping fields Saved Object types should
-     * *never* use `dynamic: true`.
+     * *never* use `dynamic: 'true'`.
      */
-    dynamic?: false | 'strict';
+    dynamic?: MappingDynamicMapping;
```

Comment:
This type is defined as:
`export type MappingDynamicMapping = boolean | 'strict' | 'runtime' | 'true' | 'false';` 

And we want to avoid anyone using `boolean` and `'true'` not sure if `'runtime'` would even work with saved objects.

So I would maybe rather type it as `'false' | 'strict'`

Reply:
It's a good point! Addressed with https://github.com/elastic/kibana/pull/226988/commits/aa30c1b348b098284b7b4cb74dd0b052a59099af